styles.css

```diff
@@ -214,120 +221,124 @@ div a img {
   background: white;
   border-top-right-radius: 100px;
+  display: grid;
+  grid-template-columns: 40% 60%;
+  padding: 100px 100px 300px 90px;
+}
+
+.about-section .wrap-one {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 20px;
 }
 
 .about-section h2 {
   font-size: 40px;
   font-weight: 700;
-  line-height: 52px;
-  letter-spacing: 0;
-  text-align: left;
-  color: #344563;
+  color: #091e42;
 }
 
 .about-section p {
   font-size: 16px;
   font-weight: 400;
-  line-height: 24px;
-  letter-spacing: 0;
-  text-align: left;
+  color: #344563;
 }
 
-.about-socials {
+.wrap-one .about-socials {
   display: flex;
-  list-style: none;
-  justify-content: space-around;
-  width: 40%;
+  gap: 20px;
+  list-style-type: none;
+  width: 50%;
 }
 
 .about-section h3 {
-  color: #6070ff;
-  font-size: 16px;
+  font-size: 17px;
   font-weight: 500;
+  color: #7f8cff;
 }
 
-.about-button {
-  margin-top: 40px;
+.wrap-one .about-btn {
+  margin-top: 10px;
 }
 
-.about-button a {
+.wrap-one a {
+  text-decoration: none;
+  padding: 10px 0 10px 0;
+  border: 1px solid #396df2;
   font-size: 17px;
   font-weight: 500;
-  line-height: 24px;
-  letter-spacing: 0.03em;
+  color: #396df2;
   text-align: center;
-  color: #6070ff;
-  border: 1.5px solid #6070ff;
-  border-radius: 16px;
-  padding: 16px;
+  border-radius: 8px;
 }
 
-.about-language {
-  margin-top: 70px;
-  list-style: none;
+.wrap-one a:hover {
+  background: #2230d2;
+  color: white;
 }
 
-.about-language ul {
-  margin-top: 40px;
+.wrap-two {
+  margin-top: 30px;
 }
 
-.about-language p {
-  background: #e5e5e5;
-  font-size: 15px;
-  font-weight: 500;
+.wrap-two .about-language {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
 }
 
-.about-language .language-arrow {
-  border-bottom: 1px solid #dfe1e6;
-  padding: 20px 0;
+hr {
+  height: 1px;
+  background-color: #dfe1e6;
+  border: none;
 }
 
-.about-language li span {
-  font-size: 20px;
-  font-weight: 500;
-  line-height: 24px;
-  letter-spacing: 0;
-  text-align: left;
-  color: #344563;
-  list-style: none;
+.about-language span {
+  width: 100%;
 }
 
-.about-language .language-items {
+.wrap-two .language-items {
+  margin-top: 30px;
   display: flex;
-  flex-direction: column;
+  justify-content: left;
+  list-style-type: none;
   gap: 10px;
 }
 
-ul .language-item img {
-  margin: 5px 15px;
-}
-
 .language-items .language-item {
-  display: flex;
-  background: #e5e5e5;
-  border-radius: 8px;
+  padding: 10px 40px 10px 40px;
+  background-color: #f7f7f9;
+  text-align: center;
 }
 
+/* =========================Contact Section=================== */
+
 .contact-container {
   background-image: url(/images/contact_form_bg.svg);
   background-repeat: no-repeat;
   background-size: 40vw;
   right: 80px;
   background-color: #6070ff;
-  margin: 114px 24px 0 24px;
+  margin-top: -10%;
   padding: 114px 24px;
   border-top-left-radius: 100px;
   background-position-x: right;
 }
```

Comment:
[Optional] Please consider changing the `background-image`  property's value to `images/contact_form_bg.svg` , currently the background-image for this container is not displaying.  You can also consider making the background image for this section appear the way it is on the Figma template.

```CSS
.contact-container {
  background-image: url(images/contact_form_bg.svg);
  background-repeat: no-repeat;
  background-size: 40vw;
  right: 80px;
  background-color: #6070ff;
  margin: 114px 24px 0 24px;
  margin-top: -10%;
  padding: 114px 24px;
  border-top-left-radius: 100px;
  background-position-x: right;
}
```

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/193124980-339b6c2e-73b4-46f9-91b9-6f5600857614.png" width=400 />

## Your Implementation

<img src="https://user-images.githubusercontent.com/16392046/193125293-f0db08de-3255-4bfa-bc43-9f36fc73e650.png" width=400 />